### PR TITLE
Story/FCT2-21088- Ui style update

### PR DESCRIPTION
**Pull request by renjithabby**
image

# PR checklist

Tick what applies before you raise. Delete anything that doesn't.

## Correctness
- [x] Does what the ticket asks for
- [x] Handles the edge cases (empty, null, zero, large inputs), not just the happy path
- [x] Errors are handled, not swallowed
- [x] New logic has tests, including the failure cases
- [x] Tests assert something real, not just run the code

## Keeping it simple
- [x] Doesn't rebuild something that already exists in the codebase
- [x] No more complex or slower than it needs to be
- [x] No leftover debug logging or commented-out code

## Easy to miss (a green pipeline won't flag these)
- [x] One logical change, not a pile of unrelated stuff

### If you touched the UI
- [x] API changes are reflected in the MSW handlers in src/mocks and the matching zod schema, so the mocked tests aren't passing against a contract that's gone
- [x] New config or feature flags are wired into the VITE build variables, not just added in code
- [x] Data fetches handle the loading, empty and error states, not just success, and match how the rest of the app already does it
- [x] Ran prettier (npm run prettier:format), since CI lints but doesn't check formatting
- [x] Checked accessibility (keyboard nav, screen reader, sensible labels), since the Lighthouse run is manual and won't flag it for you

### CODE REVIEW

File: ui-spa/src/components/govuk/LinkButton.module.scss [hunk 1]
```diff
@@ -5,6 +5,8 @@
   border-width: 0;
   cursor: pointer;
   text-decoration: underline;
+  text-decoration-thickness: max(1px, 0.0625rem);
+  text-underline-offset: 0.1578rem;
```
Comment (HasanCPS) [inline ui-spa/src/components/govuk/LinkButton.module.scss:9]: ### LinkButton underline changes are global — scope creep beyond the ticket

The `text-decoration-thickness` and `text-underline-offset` additions apply to **every** `LinkButton` across the app (14+ components use it), not just the folder path buttons. The ticket only asks for a color change on folder path link buttons. These underline tweaks might be the right GOV.UK alignment, but they should either:
- be scoped to `FolderPath.module.scss` if they're only meant for the folder breadcrumb, or
- be covered by a separate ticket / explicitly called out in the PR description if the intent is to align all LinkButtons to the GOV.UK link style.

Without that context, as a reviewer I can't tell if this is intentional or accidental.

```diff
 # Current change (global — affects all LinkButtons):
   text-decoration: underline;
+  text-decoration-thickness: max(1px, 0.0625rem);
+  text-underline-offset: 0.1578rem;
   font-size: 1rem;
```

```diff
 # If scoping to FolderPath only, move to FolderPath.module.scss instead:
   & button {
     overflow-wrap: break-word;
     word-break: break-all;
     text-align: left;
     color: #0b0c0c;
+    text-decoration-thickness: max(1px, 0.0625rem);
+    text-underline-offset: 0.1578rem;
   }
```

**Review by HasanCPS — APPROVED** [review]
LGTM!
